components/remote_settings/src/lib.rs

```diff
@@ -51,6 +57,78 @@ impl RemoteSettings {
     }
 }
 
+pub struct RemoteSettingsService {
+    config: RemoteSettingsConfig2,
+}
+
+impl RemoteSettingsService {
+    pub fn new(config: RemoteSettingsConfig2) -> Self {
+        RemoteSettingsService { config }
+    }
+
+    pub fn make_client(&self, collection_name: String) -> Result<Arc<RemoteSettingsClient>> {
+        let storage = Storage::new(
+            &Utf8Path::new(&self.config.storage_dir).join(format!("{}.sqlite", collection_name)),
+        )?;
+        let client = Client::new_for_remote_settings_service(
+            self.config.server.clone(),
+            self.config
+                .bucket_name
+                .clone()
+                .unwrap_or_else(|| "main".to_string()),
+            collection_name,
+        )?;
+        Ok(Arc::new(RemoteSettingsClient { client, storage }))
+    }
+}
+
+pub struct RemoteSettingsClient {
+    client: Client,
```

Comment:
nit: it could be called `httpClient`

Reply:
I'm going to go with `api_client`, since it does a bit more than just straight HTTP.